src/main/java/org/openpnp/model/BoardLocation.java

```diff
@@ -29,6 +31,8 @@
 public class BoardLocation extends AbstractModelObject {
 	@Element
 	private Location location;
+	@Element(required=false) // for compatility
```

Comment:
> Is there a reason you put this on BoardLocation instead of Board? The
> dimensions of a Board are fixed to the board. Do you intend to use it for
> something else?
No, i have wanted use C/Z as temp variables during assembly, that is
why i have used
Location instead of Point, but it don't matter here.

my reason:
Job -> BoardLocation -> Board -> Placements
Board = Hold placement inside boad without knowing the position of board itself.
BoardLocation = Location and dimension for Boards.

Further value is inserted into BoardLocation Table.
If having Panels, that refer to the same Board (you don't have it),
Table and Data are inconsistent as it changes more row as the code
tell the UI to update.